dapp/src/utility/proposal.utility.ts

```diff
@@ -250,3 +252,52 @@ export const checkProposalResult = (record: ProposalTableData) => {
 	}
 	return false;
 };
+
+/**
+ * Opt ASA to given account
+ * @param from : Account address that wants to opt asset
+ * @param assetID : Asset ID
+ * @param lsig : Logic Signature
+ * @param webMode : webmode instace to execute transaction
+ */
+export const optInASA = async (from: string, assetID: number, lsig: LogicSigAccount, webMode: any) => {
+	const execParam: types.ExecParams[] = [
+		{
+			type: types.TransactionType.OptInASA,
+			sign: types.SignType.LogicSignature,
+			fromAccountAddr: from,
+			lsig: lsig,
+			assetID: assetID,
+			payFlags: { totalFee: 1000 },
+		},
+	];
+
+	await toRaw(webMode).executeTx(execParam);
+}
+
+/**
+ * Transfer ASA from one account to another account
+ * @param from : Account address that wants to send transfer ASA
+ * @param to : Account address that wants to receive ASA
+ * @param numberOfASA : Number of ASA that has to be transferred
+ * @param assetID : Asset ID
+ * @param webMode : webmode instace to execute transaction
+ */
+export const transferASA = async (from: string, to: string, numberOfASA: number, assetID: number, webMode: any) => {
```

Comment:
I don't think we would require this function

Reply:
Removed